gong/errors.go

```diff
@@ -0,0 +1,17 @@
+package gong
+
+import (
+	"errors"
+)
+
+var (
+	ErrMissingClient = errors.New("JSON http client not set")
+	ErrNotArray      = errors.New("results data is not an array")
+	ErrNotObject     = errors.New("record is not an object")
+	ErrNotString     = errors.New("next is not a string")
+	ErrGeneral       = errors.New("general error")
```

Comment:
`ErrGeneral` and `ErrNotString` doesn't seem used anywhere. (Also, we want to avoid unhelpful errors like `ErrGeneral`)

Reply:
fixed